src/agent/configuration_parser/include/configuration_parser.hpp

```diff
@@ -16,74 +16,13 @@ namespace configuration
         toml::value tbl;
         configurationGithub::ConfigurationGithubModule configGithub;
 
-        void LoadGithubConfig()
```

Comment:
Since we are separating header and source file in this commit, could we do it in the first commit these files are presented? This will lead to a cleaner git history.